### DISCUSSION TITLE
How to draw curved lines/arrows?

### DISCUSSION THREAD

**Discussion by xjvs** [excalidraw/excalidraw#3926] [Q&A] [answered]
I've been searching and watching some videos, but failed to figure out how to draw curved lines/arrows, seem there should be a hand pointer appears when move to the border, but it doesn't show up for me, can someone help?
A gif is attached:
![excalidraw](https://user-images.githubusercontent.com/7976254/130251361-a5d26f12-f9bb-4bbf-bbc0-541458f4b45e.gif)

**Comment by dwelle** [comment, marked as answer]
Hey @xjvs — instead of click-dragging, use a click-click-click workflow.

At some point we will improve how you can create multi-point arrows so it's more intuitive.

**Reply by ashupk** [reply]
Please improve this

**Reply by mte0** [reply]
fr

**Comment by vickey-dev** [comment]
Please check this out for [Line/arrow editing](https://twitter.com/excalidraw/status/1267392073298251782?lang=en)

**Reply by ashupk** [reply]
This works great! Thanks :)

**Comment by Aqueuse** [comment]
I was so confused on how to do it by reading the help (click click click but where ?). Could it be possible to be more precise on the help ? I have only managed to do it by chance and 10 minutes trial /o\

**Reply by thierryc** [reply]
<img width="513" alt="image" src="https://github.com/excalidraw/excalidraw/assets/197471/d7a7f752-330d-4e77-850f-cc66a942a634">

Draw curved lines/arrows ?
In the middle you have a new handle.

**Reply by timuptocreate** [reply]
simple stuff, but ive been stuck too, up until comment of Zsolts himself in YT 
1. Already have arr/lin -> ctrl+enter (mac cmd enter)
2. Creating New -> pick arr after click once (but not move mouse !!!) after you could go with clicks.

**Reply by HaFred** [reply]
Not working T T

**Reply by HaFred** [reply]
Oh I found out the reason. If an arrow is initiated directly then it's straight and sharp, but if it's connected with the boxes for starter then it can be curved.

**Comment by leomosia** [comment]
The is an instruction on the, don't drag just click click 
![Screenshot 2024-09-03 124556](https://github.com/user-attachments/assets/e3f00de9-2cb4-4392-a0eb-1e7ff80dca11)

**Reply by usernameAshu** [reply]
This is the most clear and precise instruction to draw curved arrows using clicks.

**Comment by AbhaySolanki007** [comment]
Instead of dragging to create a straight line, click on the canvas to place the first point of your line. Continue clicking on the canvas to add subsequent points, which will form the vertices of your line. Do not drag the mouse during this step.
Enter Line Editor Mode:
Once you have placed all the necessary points for your line, select the line with the selection tool. Then, press Ctrl + Enter (or Cmd + Enter on Mac). This will activate the line editor mode, displaying multiple control points along your line.
